docs-client/src/components/Description/Mermaid/index.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect } from 'react';
+// @ts-ignore
```

Comment:
Ok. I'll try 👍

Reply:
Good!. `@types/mermaid` is better then `mermaid`.
It will be changed like this
```
const Mermaid: React.FunctionComponent<MermaidProps> = ({ chart }) => {
  useEffect(() => {
    mermaid.initialize({
      theme: Theme.Base,
      logLevel: LogLevel.Fatal,
      securityLevel: SecurityLevel.Strict,
      startOnLoad: true,
      arrowMarkerAbsolute: false,
    });

    mermaid.contentLoaded();
  });

  if (!chart) {
    return <div />;
  }

  return <div className="mermaid">{chart}</div>;
};